pkg/og/convert/pprof/profile.go

```diff
@@ -82,6 +82,14 @@ func (p *RawProfile) ParseToPprof(_ context.Context, md ingestion.Metadata) (res
 	return
 }
 
+func (p *RawProfile) isDotnetspy(md ingestion.Metadata) bool {
+	if md.SpyName == "dotnetspy" {
+		return true
+	}
+	stc := p.getSampleTypes()
+	return stc != nil && stc["inuse-space"] != nil
```

Comment:
try to detect old dotnetspy. afaik "inuse-space" was used only in dotnet.

Reply:
maybe I should only use inuse-space if SPyName==""

Reply:
I think the SpyName == ""  check might be useful.

Reply:
added